project/plugins.sbt

```diff
@@ -1,15 +1,28 @@
-// common
+// scala style formatter
 addSbtPlugin("com.github.jkinkead" % "sbt-scalariform" % "0.1.6")
+
+// to generate dependency tree based upon the library dependencies
 addSbtPlugin("com.gilt" % "sbt-dependency-graph-sugar" % "0.7.5-1")
+
+// to automatically add header descriptions to files
 addSbtPlugin("de.heikoseeberger" % "sbt-header" % "1.6.0")
 
+// to bump the version numbers and automate the release process
+addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.3")
+
+// assembly plugin to create fat jars
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.1")
+
```

Comment:
Oh, I see. Yes, I remembered I tried to use it for having stand alone executable on the servers from my home directory but building the fat jar was so slow and it was so huge that I found it useless after all.